back_end/src/test/java/com/software/grey/security/OAuth2LoginSuccessHandlerTest.java

```diff
@@ -27,8 +27,8 @@ void GoogleCorrectUser() {
         userService.saveGoogleUser(myUser);
         GoogleUser user = googleUserRepo.findByUsername("mockUser");
 
-        assert(Objects.equals(user.getUsername(), "mockUser"));
-        assert(Objects.equals(user.getEmail(), "mockEmail@gmail.com"));
-        assert(user.getRole() == Role.USER);
+        assertThat(user.getUsername()).isEqualTo("mockUser");
+        assertThat(user.getEmail()).isEqualTo("mockEmail@gmail.com");
+        assertThat(user.getRole()).isEqualTo(Role.USER);
```

Comment:
does it make any change to put the constant string before isEqualTo 
in case of user.getUsername or other getters thrown exception